firestore/angular-rewrite/src/app/restuarant-page/restuarant-page.component.css

```diff
@@ -24,6 +24,16 @@ h1 {
   display: inline;
 }
 
+button {
```

Comment:
Is this style scoped to just the restaurant page? If it affects more than one _type_ of button we might consider specifying a class to target specific buttons

Reply:
In this case, these styles apply only to the floating action button in the `restaurant-page.component.html` file. This is the only button of it's type (an elevated circular button like [this](https://m2.material.io/components/buttons-floating-action-button)), but I'm open to giving it a specific class name since it's not explicit that I'm specifically styling a floating action button here.